test/e2e/project-create.e2e.ts

```diff
@@ -31,8 +31,6 @@ test.describe('Project create', () => {
 
   test('shows field-level validation error and does not POST', async ({ page }) => {
     const input = page.getByRole('textbox', { name: 'Name' })
-    await input.pressSequentially('no sPoNgEbOb_CaSe or spaces')
-    await expect(input).toHaveValue('no-spongebob-case-or-spaces')
```

Comment:
Would be good to add a couple of types of bad input here and assert about the errors. I think we have different messages depending on what's wrong. There was one case removed when the normalization was added, but might as well do 2 or 3.

https://github.com/oxidecomputer/console/pull/2512/files#diff-0b899f6668600830b3fd43d75fa1fabec22eb556702ad569b281c8a9e7b8c47f